db/migrate/20221019161830_add_user_to_likes.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :user, null: false, foreign_key: true
+  end
+end
```

Comment:
- [ ] Again, the reference name should be `author`. Please refer to the above similar comment for the other migration. :pray:

Reply:
well resolved! ✔️